OU-151: Spike, Perses evaluation

zhuje opened the pull request:
**Summary** 
Creation of a Perses dashboard with metrics pulled from an in-cluster Prometheus.

**Related Issue** 
https://issues.redhat.com/browse/OU-151

<img width="1440" alt="Screenshot 2023-03-23 at 2 06 16 PM" src="https://user-images.githubusercontent.com/59589720/227304797-a661be7f-a048-4ff7-a10c-b50c2bac00a5.png">

kyoto commented on web/yarn-error.log:
I guess this file was added accidentally?

zhuje replied:
Removed.

zhuje committed:
OU-151 Spike, Perses evalution
Creation of a Perses dashboard with metrics pulled from an in-cluster Prometheus.

jgbernalp commented:
LGTM
